### ISSUE TITLE
Release of image classification pre-trained weights

### ISSUE THREAD

**Issue by legel** [facebookresearch/dinov2#297]
I'm working on a plant species classification application and have meanwhile been finding great value in the DinoV2 architecture and quality of code.

In the paper, there are reported metrics for several distillations (ViT-S, ViT-B, ViT-L, ViT-g) on the iNaturalist 2021 dataset.  I am specifically very interested in working with the ViT-L pre-trained model which achieves 85.1% accuracy.  (I'll be retraining it to classify over 12,500 species prevalent across the United States, for a garden design application.)

Please let me know if it would be possible to share or upload pre-trained weights / classifier head at least for ViT-L iNaturalist 2021.

**Comment by qasfb** [comment]
Thanks for your interest. The backbone models are actually not SSL-pretrained on iNaturalist, we are simply applying a linear classifier on top of the standard DINOv2 models for evaluation purposes. I expect we will not release iNat classifiers, but it should not impact you, given that you were planning to train a classifier on your side.

**Comment by patricklabatut** [comment]
> if it would be possible to share or upload pre-trained weights / classifier head at least for ViT-L iNaturalist 2021.

As @qasfb mentioned above, unless there was a significant demand, we don't expect to release such classification heads.

> there are reported metrics for several distillations (ViT-S, ViT-B, ViT-L, ViT-g) on the iNaturalist 2021 dataset.

Nevertheless, one should be able to reproduce the iNaturalist results from Table 7 by adapting and running the [linear evaluation code](https://github.com/facebookresearch/dinov2/blob/main/dinov2/eval/linear.py) on that dataset.
